### PR TITLE
#5_Sep_10_Cache_IConfiguration_IWebHostEnvironment

### PR DESCRIPTION
**Pull request by REtangler**

### CODE REVIEW

File: AltexFood Delivery/AltexFood Delivery/Controllers/ProductMvcController.cs [hunk 1]
```diff
@@ -30,6 +41,7 @@ public IActionResult GetProduct(int id)
         [HttpGet("add")]
         public IActionResult Create()
         {
+            throw new Exception("Critical failure, the PC will reboot now!");
```
Comment (yevhenbilushko) [inline AltexFood Delivery/AltexFood Delivery/Controllers/ProductMvcController.cs:44]: lets remove it)
no need to commit this

File: AltexFood Delivery/AltexFood Delivery/Controllers/ProductMvcController.cs [hunk 2]
```diff
@@ -30,6 +41,7 @@ public IActionResult GetProduct(int id)
         [HttpGet("add")]
```
Comment (yevhenbilushko) [inline AltexFood Delivery/AltexFood Delivery/Controllers/ProductMvcController.cs:41]: [HttpGet("add")]
->
[HttpGet]

or rename method to "Add"

**Comment by yevhenbilushko** [comment]
where connection string ?
seems like it should be in appsettings.json

**Review by yevhenbilushko — APPROVED** [review]
fix amd merge

File: AltexFood Delivery/AltexFood Delivery/Controllers/ProductMvcController.cs (hunk 2 above)
Comment (REtangler) [reply AltexFood Delivery/AltexFood Delivery/Controllers/ProductMvcController.cs:41]: Unfortunately, it seems that OpenAPI needs unique combination of attributes and method names, so i can't have two `[HttpGet]` attributes in same controller.
Renaming method didn't help.

Simple Get View methods don't require attributes at all if not for OpenAPI, as far as i know, but here it seems necessary.

**Pull request merged**